test/Project.toml

```diff
@@ -1,4 +1,5 @@
 [deps]
+CSV = "336ed68f-0bac-5ca0-87d4-7b16caf5d00b"
```

Comment:
CSV is only added to the test environment. Is this the intention?

Reply:
yes because there is a function to write to a csv (but no way to read it back) if the user really wants to read the results in csv form, but it shouldn't be a dependency in PowerSimulations, but it should be in the test environment for that function